x-pack/plugins/ml/public/alerting/config_validator.tsx

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useMemo } from 'react';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiCallOut, EuiSpacer } from '@elastic/eui';
+import { parseInterval, resolveBucketSpanInSeconds } from '../../common/util/parse_interval';
+import { CombinedJobWithStats, JobId } from '../../common/types/anomaly_detection_jobs';
+import { DATAFEED_STATE, JOB_STATE } from '../../common/constants/states';
+
+interface ConfigValidatorProps {
+  alertInterval: string;
+  jobConfigs: CombinedJobWithStats[];
+}
+
+/**
+ * Validated alert configuration
+ */
+export const ConfigValidator: FC<ConfigValidatorProps> = React.memo(
+  ({ jobConfigs = [], alertInterval }) => {
+    const resultBucketSpanInSeconds = useMemo(
+      () => resolveBucketSpanInSeconds(jobConfigs.map((v) => v.analysis_config.bucket_span)),
+      [jobConfigs]
+    );
+
+    const resultBucketSpanString =
+      resultBucketSpanInSeconds % 60 === 0
+        ? `${resultBucketSpanInSeconds / 60}m`
+        : `${resultBucketSpanInSeconds}s`;
+
+    if (jobConfigs.length === 0) return null;
+
+    const alertIntervalInSeconds = parseInterval(alertInterval)!.asSeconds();
+
+    const isAlertIntervalTooHigh = resultBucketSpanInSeconds < alertIntervalInSeconds;
+
+    const jobIssues = jobConfigs.reduce(
+      (acc, job) => {
+        if (job.state === JOB_STATE.FAILED) {
+          acc.failedJobsIds.push(job.job_id);
+        }
+        if ([JOB_STATE.CLOSING, JOB_STATE.CLOSED].includes(job.state)) {
+          acc.closedJobIds.push(job.job_id);
+        }
+        if ([DATAFEED_STATE.STOPPING, DATAFEED_STATE.STOPPED].includes(job.datafeed_config.state)) {
```

Comment:
should we also check for `starting`?
i believe a job could go from `starting` to `failed` so perhaps we should restrict it as we don't know for sure yet that the job is ok.